brave/brave-core
# Migrate BraveViewHostMsg_* legacy IPC messages to mojo #15546

These are the only messages present in common/render_messages.h, and
also the reason why we define a "random" value for IPC_MESSAGE_START
in that file, so let's migrate them to mojo and fix two problems at
once, since legacy IPC is deprecated anyway.

Resolves https://github.com/brave/brave-browser/issues/15546

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open the browser and go to "Settings" > "Shields"
2. Enable the "Block Scripts" option, that will block scripts globally on any URL
3. Navigate to https://coveryourtracks.eff.org and open the Brave Shields panel on the address bar
4. A non-zero number should be shown under the "Scripts blocked" section
